.github/workflows/api-code-quality.yml

```
@@ -55,9 +59,8 @@ jobs:
       - name: Set up Ruby
         uses: ruby/setup-ruby@v1
         with:
-          # .rubocop.yml の AllCops.TargetRubyVersion(3.0) に合わせるのが基本
-          # .ruby-version を使うなら：ruby-version-file: 'api/.ruby-version'
-          ruby-version: '3.0'
+          # api/.ruby-version を唯一の指定元にする（Gemfileの ruby '3.0.7' と一致）
+          ruby-version-file: 'api/.ruby-version'
```

actionlint (1.7.12)

[error] input "ruby-version-file" is not defined in action "ruby/setup-ruby@v1". available inputs are "bundler", "bundler-cache", "cache-version", "ruby-version", "rubygems", "self-hosted", "token", "windows-toolchain", "working-directory"

(action)